services/docker/docker-size.tester.js

```diff
@@ -42,5 +42,5 @@ t.create('docker image size (invalid, unknown repository)')
   .get('/_/not-a-real-repo.json')
   .expectBadge({
     label: 'image size',
-    message: 'image or tag not found',
+    message: 'invalid response data',
```

Comment:
This is indicative of a schema issue, probably the schema being too strict. Unfortunately it looks like the upstream API returns an http 200 response with an empty results key, so we'll have to allow for `results` to be empty, and then check for that use case in the `transform` function (i.e. throw a `NotFound` error)

```json
{
  "count": 0,
  "next": null,
  "previous": null,
  "results": [],
}
```

Reply:
Understood and resolved.